ccmlib/node.py

```diff
@@ -1138,6 +1138,12 @@ def __clean_bat(self):
         common.replace_in_file(bat_file, run_pattern, "wmic process call create \"" + run_text + "\" > \"" +
                                self.get_path() + "/dirty_pid.tmp\"\n")
 
+        # On Windows, remove the VerifyPorts check from cassandra.ps1
+        common.replace_in_file(os.path.join(self.get_path(), 'bin', 'cassandra.ps1'), '        VerifyPortsAreAvailable', '')
```

Comment:
We need ccm to support C* 1.2 and 2.0 for driver testing under Windows, like it used to.